pages/review.tsx

```diff
@@ -129,17 +129,30 @@ function ApprovalList({ type, category }: { type: RequestType; category: ReviewF
       {requests.map((requestObj: any) => (
         <Box sx={{ px: 3 }} key={requestObj._id}>
           <Grid container spacing={1} sx={requestObj.approvalType === 'Manager' ? managerStyling : reviewerStyling}>
-            <Grid item xs={12} md={6} lg={7}>
+            <Grid item xs={12} md={9}>
               {type === 'Upload' && (
                 <>
-                  <Link href={`/model/${requestObj.version?.model?.uuid}`} passHref>
-                    <MuiLink
-                      variant='h5'
-                      sx={{ fontWeight: '500', textDecoration: 'none', color: theme.palette.secondary.main }}
-                    >
-                      {requestObj.version?.metadata?.highLevelDetails?.name}
-                    </MuiLink>
-                  </Link>
+                  <Stack
+                    direction={{ xs: 'column', sm: 'row' }}
+                    spacing={{ xs: 0, sm: 2 }}
+                    alignItems={{ xs: 'flex-start', sm: 'center' }}
+                    justifyContent='flex-start'
+                    sx={{
+                      [theme.breakpoints.down('sm')]: {
+                        mb: 1,
+                      },
+                    }}
+                  >
+                    <Link href={`/model/${requestObj.version?.model?.uuid}`} passHref>
+                      <MuiLink
+                        variant='h5'
+                        sx={{ fontWeight: '500', textDecoration: 'none', color: theme.palette.secondary.main }}
+                      >
+                        {requestObj.version?.metadata?.highLevelDetails?.name}
+                      </MuiLink>
+                    </Link>
+                    <Chip color='primary' label={`Version: ${requestObj.version?.version}`} size='small' />
```

Comment:
You can put this chip in the stack below and remove this new stack you've added around the model name